docs/maps/web-map.md

```diff
@@ -0,0 +1,35 @@
+---
+id: web-map
+title: <map is="web-map">
+slug: /maps/web-map
+---
+
+The HTML `<map>` element has been deployed and in use in browsers since the mid-90's. It was used to provide visually appealing yet accessible home pages, by giving the Web developer the ability to design links as invisible yet interactive polygonal areas on top of one or more images.  Use of this facility became known as "client side image maps", and was often used to provide site navigation links.
+
+In recent years, client side image maps have fallen out of wide use, due to improvements in other areas of the Web platform.  However, some sites still use image maps to provide a simple, accessible, but simultaneously visual user experience.
+
+The web-map custom element suite provides a set of proof-of-concept "[customized built-in](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements)" elements based on `<map>`, `<area>` and `<img>` that will "fall back" to a client side image map in older browsers, or in the absence of JavaScript (scripting disabled).
+
+Note that because not all modern Web browsers implement HTML's customized built-in elements, it is not recommended to use this proof-of-concept on a public Web site, as end-user confusion may be the result.
```

Comment:
This document as a whole scares me 😅. Can we make this paragraph into an alert, similar to the one at https://maps4html.org/web-map-doc/docs/layers/static-images/?

Reply:
Yeah it's a sad story!